tests/src/main/java/com/hazelcast/stabilizer/tests/map/SSLEnterpriseTest.java

```diff
@@ -0,0 +1,113 @@
+package com.hazelcast.stabilizer.tests.map;
+
+
+import com.hazelcast.core.HazelcastInstance;
+import com.hazelcast.core.IMap;
+import com.hazelcast.logging.ILogger;
+import com.hazelcast.logging.Logger;
+import com.hazelcast.stabilizer.test.TestContext;
+import com.hazelcast.stabilizer.test.TestRunner;
+import com.hazelcast.stabilizer.test.annotations.Run;
+import com.hazelcast.stabilizer.test.annotations.Setup;
+import com.hazelcast.stabilizer.test.annotations.Teardown;
+import com.hazelcast.stabilizer.test.annotations.Verify;
+import com.hazelcast.stabilizer.test.utils.ThreadSpawner;
+import com.hazelcast.stabilizer.tests.helpers.KeyLocality;
+import com.hazelcast.stabilizer.tests.helpers.KeyUtils;
+import org.apache.commons.lang3.ArrayUtils;
+
+import java.util.Arrays;
+import java.util.Random;
+
+import static com.hazelcast.stabilizer.test.utils.TestUtils.assertEqualsByteArray;
+
+/**
+ * This test provides functionality of stressing for SSL feature of Hazelcast Enterprise
+ */
+public class SSLEnterpriseTest {
+
+    private final static ILogger LOGGER = Logger.getLogger(SSLEnterpriseTest.class);
+    private static final int[] BYTE_SIZE = {16 * 1024, 32 * 1024, 64 * 1024, 128 * 1024};
+    //properties
+    public String basename = "sslEnterprise";
+    public int threadCount = 3;
+    public int keyCount = 1000;
+    public int keyLength = 5;
+    public KeyLocality keyLocality = KeyLocality.Random;
+    public int logFrequency = 500;
+
+    private TestContext testContext;
+    private HazelcastInstance targetInstance;
+    private final String[] keys = KeyUtils.generateStringKeys(keyCount, keyLength, keyLocality, targetInstance);
```

Comment:
It is working at stabilizer. I know it is a confusing but i need a keys with a range, not unique. By this way, test becomes a more stressing and avoid java heap size problem.